java/test/jmri/PackageTest.java

```diff
@@ -40,6 +40,7 @@ public static Test suite() {
         suite.addTest(jmri.PathTest.suite());
         suite.addTest(jmri.PathLengthTest.suite());
         suite.addTest(jmri.PushbuttonPacketTest.suite());
+        suite.addTest(new JUnit4TestAdapter(jmri.SignalGroup.class));
```

Comment:
This line should be referencing jmri.SignalGroupTest.class

This is the cause of the errors reported by appveyor.